.azure-pipelines/third-party-test-suites.yml

```diff
@@ -0,0 +1,369 @@
+
+    # Skip task "Download osx tracer home"
+
+    # Install the tracer latest stable release to attach the profiler to the build and test steps.
+    # The script exposes the required environment variables to the following steps
+    - task: PowerShell@2
+      displayName: Install profiler latest release
+      inputs:
+        filePath: ./.azure-pipelines/setup_tracer.ps1
+
+    - script: build.cmd BuildRunnerTool --skip
+      displayName: Build Runner tool
+
+    - script: build.cmd BuildStandaloneTool --skip
+      displayName: Build Standalone tool
+
+    - task: DeleteFiles@1
+      displayName: 'Remove unneeded files'
+      inputs:
+        Contents: |
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Tool/!(*.nupkg)
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/win-x64/home*
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/win-x86/home*
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/linux-x64/home*
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/linux-musl-x64/home*
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/linux-arm64/home*
+          $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/osx-x64/home*
+
+    - publish: $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Tool
+      displayName: Uploading runner dotnet tool artifact
+      artifact: runner-dotnet-tool
+
+    - publish: $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/win-x64
+      displayName: Uploading runner standalone win-x64 artifact
+      artifact: runner-standalone-win-x64
+
+    - publish: $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/win-x86
+      displayName: Uploading runner standalone win-x86 artifact
+      artifact: runner-standalone-win-x86
+
+    - publish: $(System.DefaultWorkingDirectory)/src/Datadog.Trace.Tools.Runner/bin/$(buildConfiguration)/Console/publish/linux-x64
+      displayName: Uploading runner standalone linux-x64 artifact
+      artifact: runner-standalone-linux-x64
+
+    # Skip task "Uploading runner standalone linux-musl-x64 artifact"
+
+    # Skip task "Uploading runner standalone linux-arm64 artifact"
+
+    # Skip task "Uploading runner standalone osx-x64 artifact"
+
+- stage: run_tests
+  dependsOn: dotnet_tool
+  jobs:
+  - job: Postgres
+    strategy:
+      matrix:
+        Ubuntu_Major5:
+          imageName: ubuntu-20.04
+          os: ubuntu
+          tagName: v5.0.7
+        Windows_Major5:
```

Comment:
Should we add Alpine + Arm64 to this matrix?

Reply:
I wasn't considering adding those because I was focused on running only the platforms that the repo was already testing on. This way I don't spend a lot of time trying to figure out how to do the setup on new platforms. Do you think the current Windows/Ubuntu testing is sufficient?

Reply:
Ah, good point. I was thinking about what _we_ run on, but yeah agreed, we definitely don't want to take on that complexity